source/_posts/2016-11-08-Vapor-setup.md

```diff
@@ -0,0 +1,368 @@
+---
+title: 6 simple steps to get your first Vapor JSON response
+date: 2016-11-08 12:42:00
+tags: vapor,setup
+authorId: olha
+categories:
+- Vapor
+---
+
+6 simple steps to setup your first Vapor JSON response. We will cover the initial setup and how to make both POST and GET with controllers and models. 
```

Comment:
There will already be a title in the blog post, so the first sentence seems unnecessary. :)